operator/pkg/resources/certmanager/type_helpers.go

```diff
@@ -792,7 +792,7 @@ func getTLSConfigValues(
 	if err != nil {
 		return nil, err
 	}
-	return commonTLS.Load(ctx, k8sClient)
+	return commonTLS.Load(ctx, k8sClient, nil)
```

Comment:
Code comment could help future me why we don't need external secret expander.

Reply:
yeah, I'm not entirely sure where we want to comment that, but the main thing is that the commonTLS stuff here is synthetically constructed via looking at the v1 cluster, it doesn't actually come from a `StaticConfiguration`, so there's literally no way for it to be an external secret. It's always going to be a `SecretKeyRef` if `commonTLS` exists at all based on the returns of this function: https://github.com/redpanda-data/redpanda-operator/blob/4941e53e9dfb13bcb7afa666dc580569e83e96c4/operator/pkg/resources/certmanager/type_helpers.go#L798